test/views/html/address/PostalInternationalAddressChoiceViewSpec.scala

```diff
@@ -34,12 +34,12 @@ class PostalInternationalAddressChoiceViewSpec extends ViewSpec {
 
   "rendering PostalInternationalAddressChoiceView" must {
     "must render the correct h1 appropriate to postal address" in {
-      val result = asDocument(view(InternationalAddressChoiceDto.form).toString)
+      val result = asDocument(view(InternationalAddressChoiceDto.form()).toString)
```

Comment:
What about the forms that are `Some`?

Reply:
test case added

Reply:
@Christian-QA can you add the error message to the assertion please, otherwise the Some and None test cases are essentially the same